src/Template/Element/logs/log_entry_annexe.ctp

```diff
@@ -49,10 +49,12 @@ $langDir = LanguagesLib::getLanguageDirection($langCode);
             } elseif ($type == 'license') {
                 echo $sentenceLink.' ➜ '.$this->Html->tag('span', $sentenceText, array('class' => 'license'));
             } else { // link
+                $this->Html->script(['jquery.scrollTo.min.js', 'sentences.logs.js'],
+                                    ['block' => 'scriptBottom']);
```

Comment:
One minor thing: if the scripts are included from here, then they will be loaded in `/users/show` where they are not needed. They are only needed in `/sentences/show` so it is better to include them from there.

Reply:
Yes, you're right.

Another thing I haven't considered in my current implementation is, that it will load the scripts if a sentence currently doesn't have any translation but the logs show linking and corresponding unlinking actions, i.e. it had a translation in the past which was deleted later.

I don't know if there are any cases like that in the production database but I'm thinking about avoiding loading the scripts in that case.

Reply:
There are many cases of sentences that don't have any translations and indeed, the script would be unnecessary for those too.

Reply:
I was more thinking of a case like this:
![example_logs](https://user-images.githubusercontent.com/6566321/68886055-aceb5400-0716-11ea-9a4c-b1ca9f930e28.png)

I've just pushed my new implementation.